core/modules/query/ValueExpr.h

```diff
@@ -245,7 +245,7 @@ class ValueExpr {
 private:
     std::string _alias;
     FactorOpVector _factorOps;
-    bool _aliasIsUserDefined; /// true if the alias was defined by the user in the select statement.
+    bool _aliasIsUserDefined=false; /// true if the alias was defined by the user in the select statement.
```

Comment:
Also, the "LSST Style Guide" requires that "Assignment operators always have spaces around them":  https://developer.lsst.io/cpp/style.html#the-following-white-space-conventions-should-be-followed